### PR TITLE
Fixes #12756: update hammer host collections to use hosts.

### PR DESCRIPTION
**Pull request by waldenraines**
http://projects.theforeman.org/issues/12756

### CODE REVIEW

**Comment by komidore64** [comment]
looks good to me

ACK

File: lib/hammer_cli_katello/associating_commands.rb [hunk 1]
```diff
@@ -65,19 +65,19 @@ class RemoveHostCollectionCommand < HammerCLIKatello::RemoveAssociatedCommand
...
       extend HammerCLIForeman::AssociatingCommands::CommandExtension
 
-      class AddContentHostCommand < HammerCLIKatello::AddAssociatedCommand
-        command_name 'add-content-host'
+      class AddHostCommand < HammerCLIKatello::AddAssociatedCommand
+        command_name 'add-host'
         associated_resource :systems
 
         success_message _("The content host has been added")
         failure_message _("Could not add content host")
```
Comment (komidore64) [inline lib/hammer_cli_katello/associating_commands.rb]: still says `content host` here.

Comment (thomasmckay) [inline lib/hammer_cli_katello/associating_commands.rb]: s/content host/host/ in messages

File: lib/hammer_cli_katello/associating_commands.rb [hunk 2]
```diff
@@ -65,19 +65,19 @@ class RemoveHostCollectionCommand < HammerCLIKatello::RemoveAssociatedCommand
...
       end
 
-      class RemoveContentHostCommand < HammerCLIKatello::RemoveAssociatedCommand
-        command_name 'remove-content-host'
+      class RemoveHostCommand < HammerCLIKatello::RemoveAssociatedCommand
+        command_name 'remove-host'
         associated_resource :systems
 
         success_message _("The content host has been removed")
```
Comment (komidore64) [inline lib/hammer_cli_katello/associating_commands.rb]: `content host`

File: lib/hammer_cli_katello/associating_commands.rb [hunk 3]
```diff
@@ -65,19 +65,19 @@ class RemoveHostCollectionCommand < HammerCLIKatello::RemoveAssociatedCommand
...
 
-    module ContentHost
+    module Host
       extend HammerCLIForeman::AssociatingCommands::CommandExtension
 
-      class AddContentHostCommand < HammerCLIKatello::AddAssociatedCommand
-        command_name 'add-content-host'
+      class AddHostCommand < HammerCLIKatello::AddAssociatedCommand
+        command_name 'add-host'
         associated_resource :systems
```
Comment (thomasmckay) [inline lib/hammer_cli_katello/associating_commands.rb]: Should this be :hosts ?

**Comment by waldenraines** [comment]
[test]

**Comment by waldenraines** [comment]
@komidore64 @thomasmckay updated.

**Comment by thomasmckay** [comment]
ACK

**Comment by komidore64** [comment]
ack

**Comment by waldenraines** [comment]
Need to wait until https://github.com/Katello/katello/pull/5658 is merged before merging this.

**Comment by jlsherrill** [comment]
Merging, thanks @waldenraines !

**Pull request merged**